src/components/Contact/ContactDetails.js

```diff
@@ -20,34 +20,45 @@ import {
   phonePropTypes,
 } from './proptypes';
 
-const ContactDetails = ({
-  contact: { phoneNumber, email, location, hours, socialMedia },
-  intl,
+const SingularContactDetails = ({
+  contact: { name, phoneNumber, email, location, hours, socialMedia },
 }) => (
-  <div className="coa-ContactDetails">
-    <SectionHeader isSerif={true}>
-      {intl.formatMessage(i18n.questionsTitle)}
-    </SectionHeader>
+  <React.Fragment>
     {/* We want to keep this component easy to read and streamlined, so we are
       avoiding complicated functions in this return.
       Consider breaking into another component and calling that here*/}
-    {email && <Email email={email} />}
+    <h3 className="coa-ContactDetails__name">{name}</h3>
     {location && <Address location={location} />}
-    {phoneNumber && <PhonesList phoneNumbers={phoneNumber} />}
     {hours && !!hours.length && <Hours hours={hours} />}
+    {phoneNumber && <PhonesList phoneNumbers={phoneNumber} />}
```

Comment:
did the order get changed in the design?

Reply:
yes